beacon-chain/sync/querier.go

```diff
@@ -201,6 +203,20 @@ func (q *Querier) run() {
 	}
 }
 
+func (q *Querier) waitForAllDepositsToBeProcessed() {
+	for {
+		processed, err := q.powchain.AreAllDepositsProcessed()
+		if err != nil {
+			queryLog.Errorf("Could not check status of deposits %v", err)
+			continue
+		}
+		if processed {
+			break
+		}
+		time.Sleep(1 * time.Second)
```

Comment:
Magic num, can it be a variable? Like some polling interval with a good variable name

Reply:
alright, done